### PR TITLE
WIP additional documentation

### PR DESCRIPTION
**Pull request by sonyagreen**
More needs to be ported from google doc, including in-line questions, which I'll add as inline comments to this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+## Setting up your Pi
 
-- Sign up on [resin.io](https://dashboard.resin.io/signup)
-- go throught the [getting started guide](http://docs.resin.io/raspberrypi/nodejs/getting-started/) and create a new RPI zero application called `boombeasticmini`
-- clone this repository to your local workspace
+- Sign up for a [resin.io](https://dashboard.resin.io/signup) account
+![The resin.io sign up form](img/SignUp.png)
+- Create a new resin.io application to manage your Boombeastic
+![The resin.io application creation form](img/CreateApp.png)
+- Add SSH key
```
Comment (sonyagreen) [inline README.md]: This needs to be expanded on - linking to a doc or whatever

File: README.md [hunk 2]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+  - More detailed information about the application creation process is available in the [resin.io getting started guide](http://docs.resin.io/raspberrypi3/nodejs/getting-started/) and create a new Raspberry Pi zero or Raspberry Pi 2/2+/3 application (depending on what Raspberry Pi you have). Name this application whatever you want, e.g. `BoombeasticMini`.
+- clone this repository to your local system and open up the Boombeastic project by pasting the following into a terminal<sup>1</sup>:
+`git clone https://github.com/resin-io-projects/boombeastic`
+`cd boombeastic`
```
Comment (sonyagreen) [inline README.md]: Nice to have: notes about what is going on here and why the cd is necessary

File: README.md [hunk 3]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
 - set these variables in the `Fleet Configuration` application side tab
```
Comment (sonyagreen) [inline README.md]: also deltas?

File: README.md [hunk 4]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
-
   - `RESIN_HOST_CONFIG_dtoverlay` = `hifiberry-dac`
```
Comment (sonyagreen) [inline README.md]: Explain what this magic incantation does?

File: README.md [hunk 5]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+![resin.io fleet configuration screen](img/FleetConfig.png)
+
+- add the _resin remote_ to your local workspace using the useful shortcut in the dashboard UI.  Copy and paste the line from the top of your application page into a terminal and run it.
```
Comment (sonyagreen) [inline README.md]: >local workspace

Is there a better term for this, that doesn't sounds so dry and jargony?

Comment (sonyagreen) [inline README.md]: >Copy and paste the line from the top of your application page into a terminal and run it.

NB: no output on success

File: README.md [hunk 6]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
-- you can use any MPD client pointing it at `<boombeasticIP>:6680` (the port can be changed using `MOPIDY_MPD_PORT` env var)
+The authenticity of host 'git.resin.io (54.165.162.194)' can't be established.
+ECDSA key fingerprint is SHA256:NfwmqnKId5cx1RWpebbEuuM87bCJbdyhzRnqFES9Nnw.
+Are you sure you want to continue connecting (yes/no)?
+```
+Don't worry!  This is normal and just a security precaution before pushing code for the first time.  Just type `yes` and press enter.  (You won't see this message for subsequent `git push` commands.)
 
-## Configure via [environment variables](https://docs.resin.io/management/env-vars/)
-Variable Name | Default | Description
+- see the magic happening, your device is getting updated over-the-air (OTA)!
```
Comment (sonyagreen) [inline README.md]: Mention the 🦄 ?

File: README.md [hunk 7]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+
+After the OTA is completed, you can test the connection by pointing your browser to `your_boombeastic_ip:80` (get the device IP from the resin dashboard).
+
+## Configuring your Boombeastic 
+Next you’re going to hook up your services and make any changes to the default variables.
+
+### Boombeastic Configuration Values
```
Comment (sonyagreen) [inline README.md]: Add a description

File: README.md [hunk 8]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+#### Youtube
+Environment Variable | Description
+------------ | -------------
 MOPIDY_YOUTUBE_ENABLED | `false` | if set `true` loads the [YouTube extension](https://github.com/mopidy/mopidy-youtube)
-PORTAL_SSID | `ResinAP` | the name of the Access Point that [wifi-connect](https://github.com/resin-io/resin-wifi-connect) spawns if no known WiFi networks are found in order to expose WiFi configuration
 
+*Note: Mopidy [has other services to use? Point to their vars?] [drives the boombeastic, so it’s a great place to OSS it up.]
+
+## Building the rig
+*Notes: See the bill of materials for images of each component part. The wood layers have numbers in their upper left corner.*
```
Comment (sonyagreen) [inline README.md]: Currently in a gdrive folder - need to add as web assets

File: README.md [hunk 9]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+Start with the screw terminal block -- they look like a tiny box with two pins. Orient the terminals so they are on the same side of the wood layer as the other components, at the edge of the wood layer (so you can access them later). Solder the pins.
+
+
+Next add the set of pins. The Adafruit set of pins we received came with an extra pin, we snapped one off the end before soldering. Check the orientation of the pins before soldering - pins should be sticking up on the same side as the the screw terminals, with the short side of the pins down into the connectors.
+
+3b. Solder the pins on the LED Backpack.  The pins should be oriented so that they are sticking up toward you when you are looking at the chip on the LED Backpack.  (You might also have to snap off extra pins here.)
+
+3c. Connect the LED matrix to the LED Backpack.  BE SURE TO DO THIS THE CORRECT WAY: https://learn.adafruit.com/adafruit-led-backpack/1-2-8x8-matrix
+
+3d. Solder the speaker wires to the speaker.
```
Comment (sonyagreen) [inline README.md]: Not in the BOM

File: README.md [hunk 10]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+
+Step 14. Now’s a good time to test the wiring, before closing it all up. Boot it up! Do you see a smiley face on the LED matrix? Does it connect to the network? Does it play music without ? If not, now’s a good time to find a wiring issue and fix it.
+
+Step 15. Bolt the layers of wood together. This may take some work to move the wires so the case can close. You can bend the pins (eg. to move the amp jumpers out of the way of the speaker).
+
+Step 16. Kick out the jams.
+Go to [resin dashboard] > application > device
+Turn on public URL.
+From the public URL dropdown, grab the link.
+Paste the link in a browser (or share it with your friends!) to start playing music.
```
Comment (sonyagreen) [inline README.md]: Does Mopidy have documentation of how to use its service we can link to?

File: README.md [hunk 11]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+
+Alternative Step 16. Orrrrrr
+* To get the device IP, go to [resin dashboard] > application > device
+* Get device IP from the Status pane
+* Go to http://your_boombeastic_ip
+
+*Note: replace your_boombeastic_ip with the actual IP*
```
Comment (sonyagreen) [inline README.md]: Same page as public URL, figure out how to reflect that with a single set of instructions.

File: README.md [hunk 12]
```diff
@@ -5,51 +5,221 @@ A Raspberry Pi based smart connected speaker based on [Mopidy](https://github.co
...
+
+#### Youtube
+Environment Variable | Description
+------------ | -------------
 MOPIDY_YOUTUBE_ENABLED | `false` | if set `true` loads the [YouTube extension](https://github.com/mopidy/mopidy-youtube)
-PORTAL_SSID | `ResinAP` | the name of the Access Point that [wifi-connect](https://github.com/resin-io/resin-wifi-connect) spawns if no known WiFi networks are found in order to expose WiFi configuration
 
+*Note: Mopidy [has other services to use? Point to their vars?] [drives the boombeastic, so it’s a great place to OSS it up.]
+
+## Building the rig
```
Comment (sonyagreen) [inline README.md]: Oh and add in these assembly images https://github.com/resin-io-projects/boombeastic/tree/master/docs/v1/mini

**Comment by curcuz** [comment]
connects to #11

File: README.md (hunk 1 above)
Comment (sonyagreen) [inline README.md]: note added to the "more detailed info" line below.

File: README.md (hunk 9 above)
Comment (sonyagreen) [inline README.md]: Fixed!

File: README.md (hunk 10 above)
Comment (sonyagreen) [inline README.md]: @mccollam I didn't see anything hella helpful here https://docs.mopidy.com

I'm willing to leave it, unless you want to dig around and find something.

File: README.md (hunk 11 above)
Comment (sonyagreen) [inline README.md]: @mccollam I don't remember what this comment means -- does it make sense to you, or shall we ignore it ever happened?

**Pull request merged**